### PR TITLE
Add info and update benefits on IDP Doc

### PR DESCRIPTION
**Pull request by chomipi88**

### CODE REVIEW

File: astro/configure-idp.md [hunk 1]
```diff
@@ -18,9 +18,15 @@ Astro supports integrations with the following IdPs:
 
 This guide provides setup steps for integrating both of these identity providers on Astro. Once you complete the integration for your organization:
 
-- Astro will be listed in your IdP's set of supported applications.
-- Workspace Admins can invite a user to a Workspace directly without that user having to separately sign up for an account first.
 - Users will automatically be authenticated to Astro if they're already logged in to your IdP.
+- Enhanced user experience as repeated logins and remembering credentials is no longer required.
```
Comment (jwitz) [inline astro/configure-idp.md:22]:
```suggestion
- Users will no longer have to repeatedly login and remember credentials for their account.
```

File: astro/configure-idp.md [hunk 2]
```diff
@@ -18,9 +18,15 @@ Astro supports integrations with the following IdPs:
...
+- Credential configuration and management is entirely owned by you (the Customer).
```
Comment (jwitz) [inline astro/configure-idp.md:23]:
```suggestion
- You will have complete ownership over credential configuration and management on Astro.
```
If an administrator is using this doc, I don't think we need to clarify the "you" here.

File: astro/configure-idp.md [hunk 3]
```diff
@@ -18,9 +18,15 @@ Astro supports integrations with the following IdPs:
...
+
+:::info
+
+Astro only supports Service Provider (SP)-initiated SSO. Users are required to login via the [Cloud UI](https://cloud.astronomer.io/)
```
Comment (jwitz) [inline astro/configure-idp.md:27]:
```suggestion
Astro only supports Service Provider (SP)-initiated SSO. Users are required to log in via the [Cloud UI](https://cloud.astronomer.io/)
```

**Review by jwitz — APPROVED** [review]
Minor copy edits, otherwise LGTM!

**Review by paolaperaza — APPROVED** [review]
@chomipi88 Minor comment here for clarity but lgtm

File: astro/configure-idp.md [hunk 4]
```diff
@@ -18,9 +18,15 @@ Astro supports integrations with the following IdPs:
...
 
-- Astro will be listed in your IdP's set of supported applications.
-- Workspace Admins can invite a user to a Workspace directly without that user having to separately sign up for an account first.
 - Users will automatically be authenticated to Astro if they're already logged in to your IdP.
+- Users will no longer have to repeatedly login and remember credentials for their account.
+- You will have complete ownership over credential configuration and management on Astro.
+
+:::info
+
+Astro only supports Service Provider (SP)-initiated SSO. Users are required to log in via the [Cloud UI](https://cloud.astronomer.io/)
```
[the remark this answers is not in the document]
Comment (chomipi88) [reply astro/configure-idp.md:27]: @paolaperaza I updated the suggestion and removed 'and cannot initiate a log in via your IdP if not already authenticated.', as its another way of saying we only support SP initiated.

**Pull request merged**